doc/command_ref.rst

```diff
@@ -442,6 +442,7 @@ Groups are virtual collections of packages. DNF keeps track of groups that the u
     List all matching groups, either among installed or available groups. If
     nothing is specified list all known groups. Options ``installed`` and ``available`` narrows down the requested list.
     Records are ordered by `display_order` tag defined in comps.xml file.
+    Provides a list of all hidden groups by passing keyword ``hidden`` instead of group name.
```

Comment:
Please do not describe keyword hidden but option --hidden. Keywords are supported but deprecated. DNF should officially support only --options. Please add also -- prefix for installed and available options line 443.